lib/modules/manager/cpanfile/extract.ts

```diff
@@ -0,0 +1,90 @@
+import is from '@sindresorhus/is';
+import { logger } from '../../../logger';
+import { newlineRegex, regEx } from '../../../util/regex';
+import { CpanDatasource } from '../../datasource/cpan';
+import { GithubTagsDatasource } from '../../datasource/github-tags';
+import * as perlVersioning from '../../versioning/perl';
+import type { PackageDependency, PackageFileContent } from '../types';
+import { extractPerlVersion, formatContent } from './common';
+
+export async function extractPackageFile(
+  content: string,
+  fileName?: string
+): Promise<PackageFileContent | null> {
+  const res: PackageFileContent = {
+    deps: [],
+  };
+  const lines = content.split(newlineRegex);
+  for (let lineNumber = 0; lineNumber < lines.length; lineNumber += 1) {
+    const line = lines[lineNumber];
+
+    const perlMatch = extractPerlVersion(line);
+    if (perlMatch) {
+      res.deps.push({
+        depName: 'Perl',
```

Comment:
are we sure to use upper case here? 
```suggestion
        depName: 'perl',
```

i think lowercase is better, because package rule matchers are case sensitive